UnityProject/Assets/Scripts/Objects/Research/Artifacts/Effects/InteractEffects/TransmuteArtifactEffect.cs

```diff
@@ -0,0 +1,65 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+using Items;
+
+namespace Systems.Research
+{
+	[CreateAssetMenu(fileName = "TransmuteArtifactEffect", menuName = "ScriptableObjects/Systems/Artifacts/TransmuteEffect")]
+	public class TransmuteArtifactEffect : InteractEffectBase
+	{
+		public GameObject[] ItemResults = null;
+
+		int inputtedItem = 0;
+
+		protected override void BareHandEffect(HandApply interaction)
+		{
+			Chat.AddWarningMsgFromServer(interaction.Performer, emptyHandMessages.PickRandom());
+		}
+
+		protected override void WrongEffect(HandApply interaction)
+		{
+			Chat.AddWarningMsgFromServer(interaction.Performer, emptyHandMessages.PickRandom());
+		}
+
+		protected override void CorrectEffect(HandApply interaction)
+		{
+			//Is correct item
+			if (despawnItemOnFeed)
+			{
```

Comment:
Lots of nesting here. Contemplate inverting the Boolean expressions and/or extracting psrt of the logic into private methods to abstract complexity.